src/main/java/com/example/buyingCurrencyService/BuyingCurrencyServiceApplication.java

```diff
@@ -0,0 +1,37 @@
+package com.example.buyingCurrencyService;
+
+import com.example.buyingCurrencyService.dao.AccountRepository;
+import com.example.buyingCurrencyService.model.Currency;
+import com.example.buyingCurrencyService.model.Account;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.List;
+
+@SpringBootApplication
+public class BuyingCurrencyServiceApplication implements CommandLineRunner {
+
+    @Autowired
```

Comment:
use the lombok with injection via constructor

[https://www.baeldung.com/spring-injection-lombok](example)